modules/seqexec/server/src/main/scala/seqexec/server/gems/GemsController.scala

```diff
@@ -124,9 +135,9 @@ object GemsController {
   }
 
   final case class GemsOn(
-                           ttgs1: Ttgs1Usage,
-                           ttgs2: Ttgs2Usage,
-                           ttgs3: Ttgs3Usage,
+                           cwfs1: Cwfs1Usage,
```

Comment:
Why did you rename these?

Reply:
It is explained in the description of the PR. The code referred to the Canopus detector as Ttgs, Cwfs and Ngs. I changed all of them to Cwfs.